src/main.py

```diff
@@ -36,7 +44,7 @@
     # Load custom extensions
 
     logger.debug("Working directory: " + str(Path.cwd()))
-    extensions = [Path(f).stem for f in glob.iglob("/app/src/extensions/*.py")]
+    extensions = [Path(f).stem for f in glob.iglob("./src/extensions/*.py")]
```

Comment:
Have you tested that the command is properly loaded with this glob in a container environment?